data/mods/Xedra_Evolved/items/inventor/armor.json

```diff
@@ -0,0 +1,389 @@
+        "max_contains_weight": "20 kg",
+        "default_magazine": "medium_battery_cell"
+      }
+    ],
+    "armor": [
+      {
+        "material": [ { "type": "aluminum", "covered_by_mat": 100, "thickness": 0.6 } ],
+        "covers": [ "leg_l", "leg_r" ],
+        "specifically_covers": [ "leg_hip_l", "leg_hip_r" ],
+        "coverage": 70,
+        "encumbrance": 10
+      }
+    ],
+    "relic_data": {
+      "passive_effects": [ { "has": "WORN", "condition": "ACTIVE", "values": [ { "value": "CARRY_WEIGHT", "add": 35000 } ] } ]
+    }
+  },
+  {
+    "id": "inventor_leg_weight_on",
+    "copy-from": "inventor_leg_weight",
+    "name": { "str": "leg weight suspension system" },
+    "type": "TOOL_ARMOR",
+    "power_draw": "5 W",
+    "revert_to": "inventor_leg_weight",
+    "use_action": { "target": "inventor_leg_weight", "msg": "You turn off the weight suspension system.", "type": "transform" }
+  },
+  {
+    "id": "inventor_jump_boots",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "jump driver" },
+    "description": "Heavily mechanized boots with small integrated battery slot inside.  Upon activation make you jump higher and farther.",
+    "weight": "1657 g",
+    "volume": "1500 ml",
+    "material": [ "steel", "copper" ],
+    "symbol": "[",
+    "looks_like": "legguard_hard",
+    "color": "light_red",
+    "flags": [ "STURDY", "OUTER" ],
+    "ammo": "battery",
+    "charges_per_use": 150,
+    "use_action": { "type": "cast_spell", "spell_id": "jump_boots_leap", "no_fail": true, "level": 0 },
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE_WELL",
+        "rigid": true,
+        "flag_restriction": [ "BATTERY_LIGHT", "BATTERY_MEDIUM" ],
+        "max_contains_volume": "20 L",
+        "max_contains_weight": "20 kg",
+        "default_magazine": "medium_battery_cell"
+      }
+    ],
+    "armor": [
+      {
+        "material": [ { "type": "aluminum", "covered_by_mat": 100, "thickness": 0.7 } ],
+        "covers": [ "leg_l", "leg_r" ],
+        "specifically_covers": [ "leg_hip_l", "leg_hip_r" ],
+        "coverage": 95,
+        "encumbrance": 7
+      }
+    ]
+  },
+  {
+    "id": "aura_force",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "force shield" },
+    "description": "Small, thin thoroid with a huge baterry case, and a belt.  You can activate it, to emit a field, that covers your whole body, and decelerate all objects that it touches.",
+    "weight": "36 g",
+    "volume": "55 ml",
+    "material": [ "steel", "copper" ],
+    "symbol": "[",
+    "ammo": [ "battery" ],
+    "color": "blue",
+    "warmth": 0,
+    "material_thickness": 3,
+    "armor": [
+      {
+        "material": [ { "type": "aluminum", "covered_by_mat": 100, "thickness": 0.7 } ],
+        "covers": [ "leg_l", "leg_r" ],
+        "specifically_covers": [ "leg_hip_l", "leg_hip_r" ],
+        "coverage": 95,
+        "encumbrance": 7
+      }
+    ],
+    "relic_data": {
```

Comment:
The relic data needs to be in the turned on item, right?

Reply:
That's the unique weirdness of the `relic_data` - for `"condition": "ACTIVE"` you should have a relic data info inert item
![image](https://user-images.githubusercontent.com/67688115/232413960-f1636da3-aa57-43a0-9306-aa6bc134eb91.png)